Multisig/UI/SwiftUI/UI Library/Overlays/Snackbar/SnackbarCenter.swift

```diff
@@ -60,9 +60,9 @@ class SnackbarCenter: ObservableObject {
         }
     }
 
-    func show(error: DetailedLocalizedError) {
```

Comment:
Could it break our requirement regarding how we display error messages?

Reply:
this seems from other branch